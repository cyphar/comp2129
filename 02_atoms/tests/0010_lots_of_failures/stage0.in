START 6 1 1
START 6 2 2
START 7 2 3
START 7 10 10
START 7 100 100
START 1 2 2
START 5 -2 1
START 5 3 b
START 5 1 4
START a
START 2 1
START
START
START 2 1
START b -2
START 5 2 10 a
START 5 2 10 -3
START 5-2 2 10-3
START 5-2 2 10
START 5 512 20
START 2 2 1
