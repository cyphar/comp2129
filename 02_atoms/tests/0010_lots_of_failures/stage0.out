Cannot Start Game

Cannot Start Game

Cannot Start Game

Cannot Start Game

Cannot Start Game

Cannot Start Game

Invalid command arguments

Invalid command arguments

Cannot Start Game

Missing Argument

Missing Argument

Missing Argument

Missing Argument

Missing Argument

Missing Argument

Too Many Arguments

Too Many Arguments

Invalid command arguments

Invalid command arguments

Game Ready
Red's Turn

Invalid Command

